_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -20,9 +20,9 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with an [explicit semantic role](#explicit-role) of either `img`, `graphics-document`, `graphics-symbol`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with an [explicit semantic role][explicit role] of either `img`, `graphics-document`, `graphics-symbol`, that is [included in the accessibility tree][].
 
-**Note:** The [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should not be included in the accessibility tree unless indicated otherwise through the use of text alternative content, an explicit WAI ARIA role, or a valid `tabindex` attribute.
+**Note:** The [SVG Accessibility API Mappings, Section 5.1.2 Including Elements in the Accessibility Tree](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should not be included in the accessibility tree unless indicated otherwise through the use of text alternative content, an explicit WAI ARIA role, or a valid `tabindex` attribute.
```

Comment:
We should describe the exceptions that could include the elements in the SVG namespace in the accessibility tree. What about this?
```suggestion
**Note:** The [SVG Accessibility API Mappings, Section 5.1.2 Including Elements in the Accessibility Tree](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should normally omitted from the accessibility tree unless indicated otherwise through the use of text alternative content (one direct non-empty child `title` element or `desc` element), an explicit WAI ARIA role (a non-empty `aria-label` attribute or `aria-roledescription` attribute or an `aria-labelledby` attribute or `aria-describedby` attribute containing valid IDREF tokens), or a valid integer `tabindex` attribute.
```

Reply:
I disagree.
I don't think this rule is the place to list all the ways to give an accessible name or an explicit role to an element.